### ISSUE TITLE
Versioned in `glm` isn't exported by the `BrionConfig.cmake`

### ISSUE THREAD

**Issue by mgeplf** [BlueBrain/Brion#301]
A `CMakeLists.txt` with
```
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 find_package(Brion REQUIRED)
```

Will result in:
```
CMake Error at CMakeLists.txt:34 (find_package):
  Found package configuration file:

    .../Brion/CMake/BrionConfig.cmake

  but it set Brion_FOUND to FALSE so package "Brion" is considered to be NOT
  FOUND.  Reason given by package:
The following imported targets are referenced, but are missing: glm
```

More info internally at: https://bbpteam.epfl.ch/project/issues/browse/BSD-178

**Comment by NadirRoGue** [comment]
Thank you for the report. This PR should fix it: https://github.com/BlueBrain/Brion/pull/302

**Comment by mgeplf** [comment]
Cool, thanks for the quick response.  Are you planning a Brion 3.3.0 release and is this part of it?

**Comment by NadirRoGue** [comment]
I think so, after I have updated the synapses to support multipopulation

**Comment by mgeplf** [comment]
Fixed by #302, afaik.
